golang/pkg/cli/config_file.go

```diff
@@ -307,6 +308,13 @@ func DisabledServiceSettings(settings *Settings) *Settings {
 func PrintInfo(settings *Settings) {
 	log.Warn().Msg("🦩🦩🦩 Use the CLI tool only for NON-PRODUCTION purpose. 🦩🦩🦩")
 
+	if !settings.SettingsFile.DisableWelcomeMessage {
+		log.Info().Msgf("The config file is located at %s, where you can turn this message off.", settings.SettingsFilePath)
+		log.Info().Msgf("If you have any questions head to our Discord - discord.gg/pZWbd4fxga ! We're happy to help!")
+		log.Info().Msgf("You can learn more about the project at docs.dyrector.io, if you found this project useful please " +
+			"give us a star - github.com/dyrector-io/dyrectorio")
```

Comment:
`https://`  before link makes it clickable I think.

Reply:
Added.